reconcile/ocm_internal_notifications/integration.py

```diff
@@ -106,6 +109,9 @@ def run(self, dry_run: bool) -> None:
 
             if not dry_run and slack_user_ids:
                 users = " ".join([f"<@{uid}>" for uid in slack_user_ids])
+                instructions = get_app_interface_custom_message(
+                    "ocm-internal-notifications-instructions"
+                )
```

Comment:
It might be nice to have a default value, even `""` in this case, if this is not defined (maybe along with a warning?)